.github/workflows/chart-test.yaml

```diff
@@ -76,3 +76,66 @@ jobs:
         run: |
           echo "Some jobs failed or were cancelled."
           exit 1
+
+  deploy-to-gke:
+    runs-on: ubuntu-latest
+    env:
+      NAMESPACE: ochami-ci
+      GC_REGISTRY: ${{ secrets.GC_PROJECT_REGION }}-docker.pkg.dev/${{ secrets.GC_PROJECT_ID }}/ochami
```

Comment:
How will we keep these up to date? The GHCR repos will track the latest tags in their upstream repos, but won't push to the GCE registries.

If we want to, for example, test unreleased builds, we could use draft PRs with temporary overrides to an alternate registry/tag pending an upstream release.

Reply:
This is another shortcoming that I hadn't gotten around to. One option is to teach the existing repos for each container image how to build and push to the GC artifact registry. That would require adding GitHub Secrets to each repo that allow a Google Cloud Build robot to build and push the images as part of CI. I am not a huge fan of this approach.

Another approach would be to mirror each microservice repo in LBNL mirrors, and teach the mirrors how to build and push to the Google Cloud registry. IIUC GitLab has a [pull mirroring](https://docs.gitlab.com/ee/user/project/repository/mirror/pull.html) feature that makes this relatively straightforward. Does GitHub have something like that?

Reply:
No, we'd have to pull from our end and run our own build job.

Reply:
OK - so one could recreate the pull mirroring behavior using a robot running in a K8s CronJob that checks for new tags in the various microservice repos, and builds/pushes the container images to the GC Artifact Registry. Or maybe that is too complicated?

Reply:
IMO the simplest approach would be to just use the original GHCR-hosted images. Do we have a use case to build them separately?

Reply:
I recall a limitation somewhere in Google Cloud that container image locations were limited to Google Cloud Artifact Repository and to DockerHub, which motivated the desire to build them separately. But I might be mixing up GKE and Google Cloud Build in that regard. It is definitely simpler to just use the GHCR images.

Reply:
Awesome, thanks! That will reduce a lot of this complexity then.